pipelines/rj_smas__disparo_pic/tasks.py

```
@@ -0,0 +1,270 @@
+    Do a dispatch in chunks (função do template disparo)
+    Fixed to not mutate original payload
+    """
+    destinations = dispatch_payload["destinations"]
+    total = len(destinations)
+    original_campaign_name = dispatch_payload["campaignName"]
+
+    dispatch_date = datetime.now(timezone("America/Sao_Paulo")).strftime("%Y-%m-%d %H:%M:%S")
+
+    if total == 0:
+        log("Total de números é igual a zero. Nenhum disparo será feito.")
+        raise Exception("No destinations to dispatch")
```

Ruff (0.14.1)

Create your own exception

(TRY002)